sftp/2.0/modules/ROOT/pages/sftp-documentation.adoc

```diff
@@ -450,6 +450,8 @@ The key part of this functionality is how to determine that a file is actually n
 
 You can also use a matcher for additional filtering of files.
 
+The *On New Or Updated File* source ignores files with empty attributes list returned. This can occurs because an SFTP server does not have MDTM enabled or returns an error during the file listing.
```

Comment:
```suggestion
The *On New Or Updated File* source ignores files that return with an empty attributes list. The file can be empty if the modification time (MDTM) is not enabled by the SFTP server or if the server returns an error when listing the file.
```

Reply:
Not sure what "during the file listing" means. I take it that the error occurs when listing the attributes, but someone should verify this. 

"This" should always modify something. I'm assuming we're talking about the empty list. 

I'm also spelling out what MDTM stands for and fixing some grammatical issues.